api/app/tests/common.py

```diff
@@ -43,15 +43,36 @@ async def __aexit__(self, *error_info):
         """ Clean up anything you need to clean up """
 
 
+class MockRequestResponse:
```

Comment:
MockRequestResponse and MockResponse seem to be identical, is it intended?

Reply:
this is moreso a question for @Sybrand , but from what I can tell MockResponse is async and MockRequestResponse isn't

Reply:
Oi - bad class names - we probably need MockAsyncResponse and MockResponse?

Reply:
Sounds good. I thought they are identical but I didn't see `async` for class methods